src/servers/main.ts

```diff
@@ -80,7 +80,9 @@ const fetchServerInformation = async (url: URL): Promise<[finalURL: URL, version
     throw new Error();
   }
 
-  return [new URL('/', convertToURL(response.url)), responseBody.version];
+  const finalEndpoint = convertToURL(response.url);
+
+  return [new URL('../..', finalEndpoint), responseBody.version];
```

Comment:
@tassoevan 
Excuse me for pointing out late, maybe there are too many `..` ?

Reply:
Well, the URL for reference is `<server-root>/api/info`.

Reply:
I tested another server url such `https://foo.bar/apps/chat/`, then the result is `https://foo.bar/apps` which losts `/chat`.

Reply:
Can confirm:

Windows Rocket.Chat.Electron 3.0.3, rocketchat URL: 'http://foo.bar/rocketchat'. DevTool shows that it makes successfull query to 'http://foo.bar/rocketchat/api/info' (HTTP 200, {version: "3.1.1", success: true}).

`console.log((new URL('../..', 'http://foo.bar/rocketchat/api/info')).href); // -> 'https://foo.bar/'`

Reply:
Oh man, you guys are right. I forgot that previously I was traversing a URL with the trailing slash. Please wait, I'll try to add some tests to assert this kind of stuff.